project_event/models/project_task.py

```diff
@@ -657,20 +657,28 @@ def info_calendar_event(self):
         return self.env['calendar.event'].\
             browse(self.reservation_event_id)
 
+    def do_clone_task_reservation(self):
```

Comment:
@lgharib I think that no need to execute get_confirmation_wizard. If we are in a request state we must change the state to option and change the equivalent calendar.event state to not confirmed